--- a/chrome/browser/web_applications/os_integration/mac/apps_folder_support.mm
+++ b/chrome/browser/web_applications/os_integration/mac/apps_folder_support.mm
@@ -33,6 +33,9 @@ namespace {
 bool g_have_localized_app_dir_name = false;
 
 base::FilePath GetLocalizableAppShortcutsSubdirName() {
+  if ((true))
+    return base::FilePath("Helium Apps.localized");
+
   static const char kChromiumAppDirName[] = "Chromium Apps.localized";
   static const char kChromeAppDirName[] = "Chrome Apps.localized";
   static const char kChromeCanaryAppDirName[] = "Chrome Canary Apps.localized";
--- a/chrome/common/chrome_features.cc
+++ b/chrome/common/chrome_features.cc
@@ -76,7 +76,7 @@ BASE_FEATURE(kAppShimNotificationAttribu
 // https://crbug.com/40276068
 BASE_FEATURE(kUseAdHocSigningForWebAppShims,
              "UseAdHocSigningForWebAppShims",
-             base::FEATURE_DISABLED_BY_DEFAULT);
+             base::FEATURE_ENABLED_BY_DEFAULT);
 #endif  // BUILDFLAG(IS_MAC)
 
 #if BUILDFLAG(IS_WIN) || BUILDFLAG(IS_MAC) || BUILDFLAG(IS_LINUX) || \
