--- a/chrome/browser/ui/cocoa/accelerators_cocoa.mm
+++ b/chrome/browser/ui/cocoa/accelerators_cocoa.mm
@@ -74,7 +74,7 @@ const struct AcceleratorMapping {
     // AcceleratorForCloseWindow().
     {IDC_CLOSE_TAB, ui::EF_COMMAND_DOWN, ui::VKEY_W},
 
-    {IDC_EMAIL_PAGE_LOCATION, ui::EF_COMMAND_DOWN | ui::EF_SHIFT_DOWN,
+    {IDC_DEV_TOOLS_INSPECT, ui::EF_COMMAND_DOWN | ui::EF_SHIFT_DOWN,
      ui::VKEY_I},
 #if BUILDFLAG(ENABLE_PRINTING)
     {IDC_BASIC_PRINT, ui::EF_COMMAND_DOWN | ui::EF_ALT_DOWN, ui::VKEY_P},
--- a/chrome/browser/ui/cocoa/share_menu_controller.mm
+++ b/chrome/browser/ui/cocoa/share_menu_controller.mm
@@ -96,12 +96,6 @@ bool CanShare() {
   // to fetch sharing services that can handle the NSURL type.
   NSArray* services = [NSSharingService
       sharingServicesForItems:@[ [NSURL URLWithString:@"https://google.com"] ]];
-  NSMenuItem* email = [[NSMenuItem alloc]
-      initWithTitle:l10n_util::GetNSString(IDS_EMAIL_LINK_MAC)
-             action:@selector(emailLink:)
-      keyEquivalent:[self keyEquivalentForMail]];
-  email.target = self;
-  [menu addItem:email];
   for (NSSharingService* service in services) {
     // Email share service causes mysterious crashes, so share directly.
     // See https://crbug.com/356643975
@@ -295,13 +289,4 @@ bool CanShare() {
   return item;
 }
 
-- (NSString*)keyEquivalentForMail {
-  ui::Accelerator accelerator;
-  bool found = GetDefaultMacAcceleratorForCommandId(IDC_EMAIL_PAGE_LOCATION,
-                                                    &accelerator);
-  DCHECK(found);
-  return GetKeyEquivalentAndModifierMaskFromAccelerator(accelerator)
-      .keyEquivalent;
-}
-
 @end
